fake-server/index.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express')
+const bodyParser = require("body-parser");
+const app = express();
+const port = 3000
+
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
+
+app.post('/', (req, res) => {
+    console.log(req.body);
+    let { username, password } = req.body || {};
+    if ((username === "james") && (password = "Password1!")) {

```

Comment:
Also you are not comparing password here, you are using a single = for assignment

Reply:
oioiooooh. That's a very nasty bug. I suppose we would have caught this if I had added a unit test. Thanks for catching this @steviej08 - makes all the difference.